README.md

```diff
@@ -43,20 +46,14 @@ wget https://github.com/dipdup-io/stone-packaging/releases/latest/download/cpu_a
 
 ### Creating and Verifying a Test Proof Using Binaries
 
-Clone the repository:
```

Comment:
The same, we still need that. Please, try to reproduce the instructions you provided